cmd/lotus-storage-miner/info_all.go

```diff
@@ -35,88 +35,88 @@ var infoAllCmd = &cli.Command{
 
 		fmt.Println("#: Version")
 		if err := lcli.VersionCmd.Action(cctx); err != nil {
```

Comment:
It would be nice to annotate these errors with _what_ failed (even just a line number).